### PR TITLE
GEOMESA-1154 - Improve KDS consumer install

### PR DESCRIPTION
**Pull request by jnh5y**

### CODE REVIEW

File: geomesa-tools/bin/install-kafka.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    NL=$'\n'
+    read -r -p "Install Kafka DataStore dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
+    confirm=${confirm,,} #lowercasing
+    if [[ $confirm =~ ^(yes|y) ]]; then
+        # get stuff
+        declare -a urls=(
+            "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/${kafka.version}/kafka-clients-${kafka.version}.jar"
+            "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka_2.11/${kafka.version}/kafka_2.11-${kafka.version}.jar"
+            "https://search.maven.org/remotecontent?filepath=com/yammer/metrics/metrics-core/2.2.0/metrics-core-2.2.0.jar"
+            "https://search.maven.org/remotecontent?filepath=com/101tec/zkclient/0.3/zkclient-0.3.jar"
```
Comment (elahrvivaz) [inline geomesa-tools/bin/install-kafka.sh]: not crazy about the mix of dynamic and static versions... but nothing better occurs to me.

File: docs/user/installation_and_configuration.rst [hunk 2]
```diff
@@ -347,7 +347,7 @@ Accumulo 1.6
 * accumulo-server-base-1.6.4.jar
 * accumulo-trace.1.6.4.jar
 * libthrift-0.9.1.jar
-* zookeeper-3.4.6.jar
+* zookeeper-3.4.5.jar
```
Comment (elahrvivaz) [inline docs/user/installation_and_configuration.rst]: any reason to roll this back? seems like we should keep 3.4.6

Comment (jnh5y) [inline docs/user/installation_and_configuration.rst]: We mention 3.4.5 elsewhere, and our main dependency is on 3.4.5.  I was just going for consistency.

Comment (jnh5y) [inline docs/user/installation_and_configuration.rst]: If we want to be consistent and move to a later version, that's good by me!

File: geomesa-tools/bin/install-kafka.sh (hunk 1 above)
Comment (jnh5y) [inline geomesa-tools/bin/install-kafka.sh]: We don't have a direct dependency on metrics-core or zkclient.

Comment (jahhulbert-ccri) [inline geomesa-tools/bin/install-kafka.sh]: but if we change kafka versions they might change? What about a conf file for all versions somewhere? can be another ticket. e.g. https://github.com/jahhulbert-ccri/cloud-local/blob/master/conf/cloud-local.conf that can be referenced by bash scripts

File: docs/user/installation_and_configuration.rst (hunk 2 above)
Comment (jahhulbert-ccri) [inline docs/user/installation_and_configuration.rst]: +1 for 3.4.6

File: geomesa-tools/bin/install-kafka.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+if [[ (-z "$1") ]]; then
+    echo "Error: Provide one arg which is the target directory (e.g. /opt/jboss/standalone/deployments/geoserver.war/WEB-INF/lib)"
+    exit
+else
+    install_dir=$1
+    NL=$'\n'
+    read -r -p "Install Kafka DataStore dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
+    confirm=${confirm,,} #lowercasing
+    if [[ $confirm =~ ^(yes|y) ]]; then
+        # get stuff
```
Comment (jahhulbert-ccri) [inline geomesa-tools/bin/install-kafka.sh]: change comment to note that the other jars (yammer, zkclient) are required for KDS to work

Comment (jnh5y) [inline geomesa-tools/bin/install-kafka.sh]: (I copied the install-hadoop-accumulo.sh file;))

I think the rst is clear.... look below.

**Comment by jahhulbert-ccri** [comment]
@jnh5y you get a chance to review the comments? overall looks pretty good.

**Comment by jahhulbert-ccri** [comment]
29b136efe82fad63b6b4e282124f3632f3810421

**Pull request closed without merging**